grammars/ambiguity.tex

```diff
@@ -2,14 +2,18 @@ \section{Ambiguity}
 
 A sentence $x$ of $G$ is ambiguous iff it admits different syntax trees. In that case $G$ is ambiguous. The degree of ambiguity of $x$ is the number of distinct trees of $x$, of $G$ is the maximum over its sentences.
 
+\paragraph{RegExp ambiguity} given an r.e. f , we number its letters and we obtain a numbered r.e. is ambiguous if the language defined by the numbered r.e. contains two distinct strings x and y that become identical when the numbers are erased.
```

Comment:
`given an r.e. f , ` please remove the space before the comma.
Can you please double check this sentence? I think there is something missing (like a preposition or so).